dbms/src/Storages/StorageMergeTree.h

```diff
@@ -132,6 +132,9 @@ class StorageMergeTree : public ext::shared_ptr_helper<StorageMergeTree>, public
 
     BackgroundProcessingPool::TaskHandle background_task_handle;
 
+    std::vector<MergeTreeData::AlterDataPartTransactionPtr> prepareAlterTransactions(
+        const ColumnsDescription & new_columns, const IndicesDescription & new_indices, const size_t thread_pool_size);
```

Comment:
Still waiting for moving the number of threads to `max_alter_threads` setting (see `Settings.h`).

Reply:
done